ingest/expression_files/dense_ingestor.py

```diff
@@ -36,18 +37,21 @@ def __init__(self, file_path, study_file_id, study_id, **matrix_kwargs):
         # To allow additional optional keyword arguments like gene_id
         self.matrix_params = matrix_kwargs
         self.gene_names = {}
-        self.header = self.set_header()
+        csv_file_handler = self.open_file(self.file_path)[0]
+        self.header = DenseIngestor.set_header(csv_file_handler)
         self.csv_file_handler, self.file_handler = self.open_file(self.file_path)
-        # Reset csv reader to first gene row
         next(self.csv_file_handler)
 
-    def set_header(self):
-        csv_file_handler = self.open_file(self.file_path)[0]
+    @staticmethod
+    def set_header(csv_file_handler) -> List[str]:
```

Comment:
@bistline will you double check this please

Reply:
I think this is good now